syntax = "proto3";

package user;

service UserService {
  rpc GetUser(UserRequest) returns (User);
}

message UserRequest {
  string user_id = 1;
}

message User {
  string user_id = 1;
  string name = 2;
  int32 level = 3;
}
